src/js/vre/utils/labels.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export const dischargeTypeLabels = {
+  honorable: 'Honorable',
+  general: 'General',
+  other: 'Other Than Honorable',
+  'bad-conduct': 'Bad Conduct',
+  dishonorable: 'Dishonorable',
+  undesirable: 'Undesirable'
+};
+
+export const benefitsLabels = {
+  chapter31: <p>Vocational Rehabilitation Benefits (Chapter 31)</p>,
+  ownServiceBenefits: <p>Veterans' Education Assistance based on your own service</p>,
```

Comment:
Curly apostrophe on this line and the next, I think.

Reply:
Thanks, fixed.